python/README.md

```diff
@@ -1,9 +1,11 @@
-# ohsome-examples
-Here you find examples and use cases for the ohsome API using jupyter notebooks. 
+# ohsome examples in Python
+Here you find examples and use cases for the ohsome API using jupyter notebooks.
 
 ## Jupyter Notebooks
+
+* [Data Aggregation Endpoint](https://nbviewer.jupyter.org/github/GIScience/ohsome-examples/blob/master/python/jupyter-notebooks/ohsome-data-aggregation.ipynb)
 * [Analysing HOT Tasking Manager Projects in Nepal](https://nbviewer.jupyter.org/github/GIScience/ohsome-examples/blob/master/python/jupyter-notebooks/ohsome_api_hot_tm_project1008.ipynb)
-* [ohsome API: Data Aggregation Endpoint](https://nbviewer.jupyter.org/github/GIScience/ohsome-examples/blob/master/python/jupyter-notebooks/ohsome-aggregation-plot.ipynb)
+* [Complex analysis with the magical filter parameter](https://nbviewer.jupyter.org/github/GIScience/ohsome-examples/blob/master/python/jupyter-notebooks/cycling-magic-filter-ohsomeAPI.ipynb)
 
 ## Documentation of Ressources:
 * [ohsome Rest-API](http://api.ohsome.org)
```

Comment:
We never called the `ohsome API` `ohsome Rest-API`. Is there a reason it is called differently here?